docs/en/operations/server_settings/settings.md

```diff
@@ -666,13 +666,13 @@ Path to the file that contains:
 
 ## zookeeper {#server-settings_zookeeper}
 
-Contains settings that allow ClickHouse to interact with [ZooKeeper](http://zookeeper.apache.org/) cluster.
+Contains settings that allow ClickHouse to interact with a [ZooKeeper](http://zookeeper.apache.org/) cluster.
 
-ClickHouse uses ZooKeeper for storing metadata of replicas when using replicated tables. If replicated tables are not used, this parameter section can be omitted.
+ClickHouse uses ZooKeeper for storing metadata of replicas when using replicated tables. If replicated tables are not used, this section of parameters can be omitted.
```

Comment:
I'd just drop "of parameters"

Reply:
I think it will be not obvious which section we mean. The `zookeeper` tag was not earlier introduced as a section of parameters.